bindings/qt/QTMegaListener.cpp

```diff
@@ -111,6 +111,22 @@ void QTMegaListener::onReloadNeeded(MegaApi *api)
     QCoreApplication::postEvent(this, event, INT_MIN);
 }
 
+void QTMegaListener::onEvent(MegaApi *api, MegaEvent *e)
+{
+    switch(e->getType())
+    {
+        case MegaEvent::EVENT_ACCOUNT_CONFIRMATION:
+        {
+            QTMegaEvent *event = new QTMegaEvent(api, (QEvent::Type)QTMegaEvent::OnEvent);
+            event->setEvent(e ? e->copy() : NULL);
```

Comment:
This doesn't make sense because if `e` is NULL the code would crash some lines above.